ipn/ipnlocal/c2n.go

```diff
@@ -25,6 +25,8 @@ import (
 	"tailscale.com/version/distro"
 )
 
+var c2nLogHeap func(http.ResponseWriter, *http.Request) // non-nil on most platforms (c2n_pprof.go)
```

Comment:
Fine, but I hate that click through doesn't work for these in vscode

Reply:
does for me?

Reply:
oh, that it jumps to this line